<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>512</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NNC Utils</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>chk_image_dup</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>441</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Target dir</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="cid_src_dir">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hash</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cid_hash">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Average hash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Perceptual hash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Perceptual hash (simple)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Difference hash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Difference hash (vertical)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wavelet hash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sha1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>md5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Hash size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="cid_hash_size">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="cid_select_src_dir">
        <property name="text">
         <string>Select target dir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="cid_proceed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>441</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Proceed</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>gen_image_dataset</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="nnc_log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>461</width>
     <height>191</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="nnc_close">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <action name="actionCIDProceed">
   <property name="text">
    <string>CIDProceed</string>
   </property>
   <property name="toolTip">
    <string>CID Proceed</string>
   </property>
  </action>
  <action name="actionCIDSelect_src_dir">
   <property name="text">
    <string>CIDSelect_src_dir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nnc_close</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cid_proceed</sender>
   <signal>pressed()</signal>
   <receiver>actionCIDProceed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cid_select_src_dir</sender>
   <signal>pressed()</signal>
   <receiver>actionCIDSelect_src_dir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
